src/components/ContentWidgets/ToDoList/ToDoList.css

```diff
@@ -0,0 +1,7 @@
+.completedListItem {
```

Comment:
CSS class names should have hyphen case, not camel case

`completed-list-item` not `completedListItem`